WordPress/Classes/ViewRelated/Reader/CommentContentView.m

```diff
@@ -219,6 +280,15 @@ - (WPRichTextView *)viewForContent
     return textContentView;
 }
 
+- (UIView *)viewForCommentMeta
+{
+    UIView *view = [[UIView alloc] initWithFrame:CGRectMake(0.0, 0.0, 100.0, CommentContentViewMetaHeight)];
```

Comment:
Hmm... Well, in this case the frame is arbitrary and really disposable (constraints are gonna do what constraints are gonna do!). I included the height constant cos I thought it added some context.  I'll add a comment about the frame being arbitrary.